fastembed/rerank/cross_encoder/onnx_text_model.py

```diff
@@ -0,0 +1,28 @@
+from typing import Sequence
+from fastembed.common.onnx_model import OnnxModel, OnnxProvider
+from optimum.onnxruntime import ORTModelForSequenceClassification
+from transformers import AutoTokenizer
```

Comment:
we don't use optimum nor transformers in fastembed, cuz they will make the size of the package enormous
instead, we use `tokenizers` and `onnxruntime`, could you please rewrite it using them?